client/src/pages/SignUp.js

```diff
@@ -63,24 +44,49 @@ const SignUp = () => {
     }
   }; // Controlled form inputs implemented here
 
-  const handleLogin = (e) => {
+  const handleSignup = (e) => {
     e.preventDefault();
-    if (step === "first") {
-      if (form.password === form.confirmPassword) setStep((prev) => "last");
-      else setError(true);
-    } else {
-      // Implement API call to send data to flask backend :: reminder
+    if (form.password === form.confirmPassword) {
+      fetch("/signin", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          email: form.email,
+          password: form.password,
+          name: form.name,
+        }),
```

Comment:
Small nitpick here, can we change the field "name" to "full_name"? Because in the back-end it requires the field called full_name and i'm not sure if it will break if the json sends a field of name instead of full_name.

Reply:
I can do that but it won't affect as you can assign the value at backend like full_name = name